components/Buttons/Buttons.ts

```diff
@@ -0,0 +1,32 @@
+import { Button, styled } from '@mui/material'
```

Comment:
For this file, there are repeated styles that defile the DRY principle, for example, padding, color, and border-radius, here is an idea of what I think would suffice, create a custom button(StyledButton) off the Button(MUI) component and add the recurring styles, then from the Custom Button, you can then make other buttons and add some customizations such as background color and hover color.

Next in the Button Component, it will be nice if we have just one button component, create something like index.tsx that receives props and from that props return the specific buttons needed, example if a button accepts props for type which can be "edit" | "submit", based on this props you can return either the edit button  component or the Signout button component (which I think should be named submit)

Lastly, the padding on the buttons is too much, please reduce it a bit.